desktop/src/main/java/bisq/desktop/main/MainView.java

```diff
@@ -167,6 +169,8 @@ public MainView(MainViewModel model,
     @Override
     protected void initialize() {
         MainView.rootContainer = root;
+        if (LanguageUtil.isDefaultLanguageRTL())
+	    MainView.rootContainer.setNodeOrientation(NodeOrientation.RIGHT_TO_LEFT);
```

Comment:
Please apply Bisq coding conventions and style guide: https://github.com/bisq-network/bisq/blob/master/CONTRIBUTING.md#style-and-coding-conventions